test/aws/mock_data.py

```diff
@@ -20,6 +21,34 @@
 TEST_CLOUDFRONT_ID = "test-cloudfrontet-TEST"
 TEST_CLOUDFRONT_DOMAIN_NAME = "test-s3.cloudfront.net"
 INSTANCE_ID = "test-instance"
+VM_BUCKET_OUTPUTS = {
+    "VMBucketId": TEST_BUCKET_ID,
+    "VMExportRoleId": TEST_ROLE_ID,
+    "CfDistributionId": TEST_CLOUDFRONT_ID,
+    "CfDistributionDomainName": TEST_CLOUDFRONT_DOMAIN_NAME,
```

Comment:
Define constants for the keys ("VMBucketId", ...) in exasol/ds/sandbox/lib/cloudformation_templates/example_data_bucket.py and re-use here.

Reply:
This finding potentially touches the heart of the problem, major parts of the implementation, refactoring, discussions, review findings, and fixes across the last 2-3 weeks. 

The ironic part is that these constants already exist. But using the constants here would make the test pointless, as the test then verifies only that a constant equals its own value. 

A better approach in my eyes would have been to not replace the output keys in the first place and leave them static, and also to use identical output keys in the CF templates for different buckts and WAFs. But this collides with the overly-specific names that had been chosen in the past for some of the outputs, e.g. 
* specific "VMBucketId" vs. "ExampleDataBucketId" 
* and generic "CfDistributionId" (identical for both buckets)

But we decided to not change the existing deployment, which I still can understand and support, as it would introduce a breaking change on the infrastructure side impacting for example on the CLI command `show-aws-assets` which relies on the exact keys of the outputs. 

I propose to have a joint discussion.